src/libcore/str/mod.rs

```diff
@@ -795,7 +764,7 @@ impl TwoWaySearcher {
     // that (u, v) is a critical factorization for the needle.
     #[inline]
     fn next(&mut self, haystack: &[u8], needle: &[u8], long_period: bool)
-            -> Option<(usize, usize)> {
+    -> Option<(usize, usize)> {
```

Comment:
I believe the current style is to align the `-` of the `->` under the first character after the `(` (in this case the `&` symbol.